### PR TITLE
chore: public api in readme

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
## Description

This PR adds mention of what is considered public api to readme. Wording sucks, because it's Friday 😅

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly. If you are changing **code related to user secrets** you need to really make sure that [security documentation](https://github.com/superfaceai/one-sdk-js/blob/main/SECURITY.md) is correct.
- [ ] I have read the [CONTRIBUTING](https://github.com/superfaceai/one-sdk-js/blob/main/CONTRIBUTING.md) document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,18 @@ If you have any questions, want to report a bug, request a feature or you just w
 
 You can find more options for reaching us on the [Support page](https://superface.ai/support).
 
+## Public API
+
+If you decide to use some function from this package in your own code you definetly can but be aware that not every part of package is version controlled.
```
Comment (tmladek) [inline README.md:136]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -131,6 +131,18 @@ If you have any questions, want to report a bug, request a feature or you just w
...
+
+Only function and API of entities below is under version control:
```
Comment (tmladek) [inline README.md:138]:
```suggestion
Only functions and APIs of entities below are a part of the public API, and can be safely relied upon not to break between semver-compatible releases.

Using other parts of this package is at your own risk.
```

File: README.md [hunk 3]
```diff
@@ -131,6 +131,18 @@ If you have any questions, want to report a bug, request a feature or you just w
...
+
+* SuperfaceClient API
+* Profile API
+* UseCase API
+* SuperJsonDocument object
```
Comment (tmladek) [inline README.md:143]:
```suggestion
* SuperJsonDocument Object
```

File: README.md (hunk 2 above)
Comment (tmladek) [reply README.md:138]: what do you think @Jakub-Vacek? :)

**Review by tmladek — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tmladek) [reply README.md:138]: rationale - "you can use this package, but beware, there are dragons somewhere!" is not really the message IMHO, it's more that "if you want to use this package, there's A, B & C and everything else is dragons"

Comment (Jakub-Vacek) [reply README.md:138]: I like it :) 
A, B & C are also dragons but we will try to keep them dragons😅

**Comment by jnv** [comment]
I think this should mention the [reference](https://superface.ai/docs/reference/one-sdk)

**Review by lukas-valenta — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -131,6 +131,20 @@ If you have any questions, want to report a bug, request a feature or you just w
...
+
+Using other parts of this package is at your own risk.
+
+* SuperfaceClient API
+* Profile API
+* UseCase API
+* SuperJsonDocument Object
+* Result API
+
+Use of public API is described in [reference](https://superface.ai/docs/reference/one-sdk).
```
Comment (tmladek) [inline README.md:146]:
```suggestion
Use of public APIs is described in the [reference](https://superface.ai/docs/reference/one-sdk).
```

**Review by tmladek — CHANGES_REQUESTED** [review]
👍

**Review by tmladek — APPROVED** [review]

**Review by jnv — APPROVED** [review]

**Pull request merged**